.github/workflows/publish.yml

```diff
@@ -0,0 +1,90 @@
+name: Build and Publish Package
+
+on:
+  create
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish_apisix:
+    name: Build and Publish RPM Package
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2.3.5
+        with:
+          submodules: recursive
+
+      - name: Extract Tags name
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
+        id: tag_env
+        shell: bash
+        run: |
+          echo "##[set-output name=version;]$(echo ${GITHUB_REF##*/})"
+
+      - name: Clone apisix-build-tools
+        run: |
+          git clone https://github.com/api7/apisix-build-tools.git
```

Comment:
We don't need to clone the repo itself?

Reply:
Yes, I forgot to remove this step as I created the PR for APISIX. Done! Thanks